.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+<!-- Please link to a related issue below. -->
+
+## Changes
+<!--- Describe your changes -->
+
+## Checklist
+- [ ] I added markup tests, or they don't apply here because...
+- [ ] I updated the changelog at `CHANGES.md`
+- [ ] My name is in `AUTHORS.txt`, under contributors
```

Comment:
```suggestion
- [ ] Added markup tests, or they don't apply here because this is a minor keyword bump...
- [ ] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors
```

Reply:
I left it blank after "because", so that the author would be forced to specify the reason. What do you think?

Reply:
Ok... `because... ` then.

Reply:
```suggestion
- [ ] Added markup tests, or they don't apply here because...
- [ ] Updated the changelog at `CHANGES.md`
- [ ] Added myself to `AUTHORS.txt`, under Contributors
```